packages/material-ui/src/styles/transitions.test.js

```diff
@@ -138,11 +138,11 @@ describe('transitions', () => {
 
     it('should return NaN when passed a negative number', () => {
       const zeroHeightDurationNegativeOne = transitions.getAutoHeightDuration(-1);
-      expect(isNaN(zeroHeightDurationNegativeOne)).to.equal(true);
+      expect(Number.isNaN(zeroHeightDurationNegativeOne)).to.equal(true);
```

Comment:
This changes behavior of the test and should be reverted unless the tested unit changed. We can handle this in another PR.

Reply:
Reverted